website/docs/components/form/layout/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,13 @@
+## Anatomy
+
+![Anatomy of formLayout](/assets/components/form/layout/form-anatomy-expanded.png)
```

Comment:
In the image, "additional content" isn't a part of the Section Header, but it should be, correct?